### ISSUE TITLE
[Bug] exporting setting as JSON triggers a prompt to restart

### ISSUE THREAD

**Issue by rori1977** [lumapu/ahoy#1671]
### Platform

ESP8266

### Assembly

I did the assebly by myself

### Antenna

circuit board

### Power Stabilization

Elko (~100uF)

### Connection picture

- [ ] I will attach/upload an Image of my wiring

### Version

0.8.123

### Github Hash

8d4ca49

### Build & Flash Method

AhoyDTU Webinstaller

### Setup

Irrelevant for this bug

### Error description

After downloading the settings, I am reminded to perform a (unneccesary) restart. Exporting to JSON should not require any restart.

**Comment by rori1977** [comment]
![Screenshot_2024-06-09-21-59-35-739_org mozilla firefox-edit](https://github.com/lumapu/ahoy/assets/56794204/d03588c1-e4bd-4199-a20b-f0e2744bcf12)

**Comment by lumapu** [comment]
yes that's true. It occurs since we save the settings prior downloading them. The 'normal' save feature requires a reboot, because the ESP does not know if settings were changed which will be applied with ne next reboot earliest. Thank you for the note.
